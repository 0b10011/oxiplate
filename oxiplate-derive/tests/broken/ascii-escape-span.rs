use oxiplate_derive::Oxiplate;

#[derive(Oxiplate)]
#[oxiplate_inline = "hello\n{{"]
struct Data {}

fn main() {
    let data = Data {};

    print!("{}", data);
}
