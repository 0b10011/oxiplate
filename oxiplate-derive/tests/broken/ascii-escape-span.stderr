error: Expected an expression.
 --> tests/broken/ascii-escape-span.rs:4:30
  |
4 | #[oxiplate_inline = "hello\n{{"]
  |                              ^
